fabric-renderer-api-v1/src/main/java/net/fabricmc/fabric/api/renderer/v1/model/ForwardingBakedModel.java

```diff
@@ -92,7 +92,7 @@ public ModelTransformation getTransformation() {
 	}
 
 	@Override
-	public ModelItemPropertyOverrideList getItemPropertyOverrides() {
-		return wrapped.getItemPropertyOverrides();
+	public ModelOverrideList getOverrides() {
```

Comment:
It seems like this API method shouldn't just be casually renamed? Should be deprecated + new one added.